test/Dialect/FIRRTL/parse-basic.fir

```diff
@@ -1111,6 +1111,19 @@ circuit MyModule :     ; CHECK: firrtl.circuit "MyModule" {
     ; CHECK-NEXT: [[NOT_COND:%.+]] = firrtl.not %cond
     ; CHECK-NEXT: firrtl.assert %clock, [[NOT_COND]], [[NOT_ENABLE]]
 
+    ; Verification Library Covers
+
+    ; Predicate modifier `noMod`
+    when not(cond):
+      printf(clock, enable, "Assertion failed: [verif-library-cover]<extraction-summary>{\"predicateModifier\":{\"type\":\"noMod\"},\"conditionalCompileToggles\":[{\"type\":\"unrOnly\"},{\"type\":\"formalOnly\"}],\"labelExts\":[\"cover\",\"label\"],\"format\":{\"type\":\"sva\"},\"baseMsg\":\"cover hello world\"}", value)
+      stop(clock, enable, 1)
```

Comment:
I think this is actually coming out of Chisel as an `assert` that is not under the `when`? Is that right?